### ISSUE TITLE
The download links in gamemaker.sh are not working

### ISSUE THREAD

**Issue by cpoliticas** [zerojay/RetroPie-Extra#338]
The download files install_bin_gamemaker function from https://github.com/zerojay/RetroPie-Extra/blob/master/scriptmodules/ports/gamemaker.sh script are broken and can't install those games in a raspberry pi.

**Comment by sprayydev** [comment]
você executou o script? vá em Retropie-Setup >> Gerenciar Pacotes >> Gerenciar pacotes experimentais >> Gamemaker

ou execute-o diretamente de um terminal:

sudo ./RetroPie-Setup/retropie_setup.sh

**Comment by cpoliticas** [comment]
> você executou o script? vá em Retropie-Setup >> Gerenciar Pacotes >> Gerenciar pacotes experimentais >> Gamemaker
> 
> ou execute-o diretamente de um terminal:
> 
> sudo ./RetroPie-Setup/retropie_setup.sh

Yes, I've done it both ways.  The problem seems to be that the links running in the script doesn't work anymore: 

- https://www.yoyogames.com/download/pi/tntbf
- https://www.yoyogames.com/download/pi/crate
- https://www.yoyogames.com/download/pi/castilla

I don't know if yoyogames moved the games to another place, or they removed them completely.

**Comment by PokeEngineer** [comment]
I'm pretty sure YoYo wiped the links off the face of the Earth, most likely because it didn't work out of the box with newer versions of Raspbian and the guy who was working on the thing at the time retired from the company.
Luckily, Wayback Machine seems to have the redirect links and the direct download links archived:

https://web.archive.org/web/20160922155812/https://www.yoyogames.com/download/pi/tntbf
https://web.archive.org/web/20160922155746/https://www.yoyogames.com/download/pi/crate
https://web.archive.org/web/20180413223436/https://www.yoyogames.com/download/pi/castilla

https://web.archive.org/web/20190618222906/http://download.yoyogames.com/pi/TheyNeedToBeFed.tar.gz
https://web.archive.org/web/20190618222852/http://download.yoyogames.com/pi/SuperCrateBox.tar.gz
https://web.archive.org/web/20180413223437/http://download.yoyogames.com/pi/MalditaCastilla.tar.gz

**Comment by 2play** [comment]
games on Buster wont run with below error:
`/usr/lib/arm-linux-gnueabihf/libcurl.so.4: version `CURL_OPENSSL_3' not found (required by RetroPie/roms/gamemaker.OFF/MalditaCastilla/MalditaCastilla)`

**Comment by sprayx** [comment]
close this.
